### PR TITLE
sync up private and regular tab partition names for extensions

### PR DESCRIPTION
**Pull request by bridiver**
@bbondy

### CODE REVIEW

**Comment by bridiver** [comment]
this is not ready to merge, there is an issue with it

File: js/components/frame.js [hunk 1]
```diff
@@ -121,9 +121,11 @@ class Frame extends ImmutableComponent {
 
     let partition
     if (this.props.frame.get('isPrivate')) {
-      partition = 'private-1'
+      partition = 'default'
     } else if (this.props.frame.get('partitionNumber')) {
       partition = `persist:partition-${this.props.frame.get('partitionNumber')}`
+    } else {
+      partition = 'persist:default'
```
Comment (bbondy) [inline js/components/frame.js]: Does this cause a different partition name for the default partition? In particular people that have a gmail account setup for example will still have that setup right?

Comment (bridiver) [inline js/components/frame.js]: is there something other than cookies that gets stored locally for the partition?

Comment (bbondy) [inline js/components/frame.js]: I think local storage, cache, cookies, etc.

Comment (bridiver) [inline js/components/frame.js]: that will definitely be lost as discussed in slack. Everyone seemed ok with it, but if not I can go another route. There is also currently an issue with ad insertion that this fixes for some reason.

Comment (bbondy) [inline js/components/frame.js]: I don't think it's ok to make people re-setup all user accounts.

Comment (bbondy) [inline js/components/frame.js]: @bridiver said he verified and it's considered the same partition. Merging.

**Pull request merged**